kubernetes/resource_kubernetes_ingress_test.go

```diff
@@ -452,47 +437,3 @@ resource "kubernetes_ingress" "test" {
   wait_for_load_balancer = true
 }`, name, name, name, name, name, name, name)
 }
-
-func testAccKubernetesIngressConfig_stateUpgradev0(provider, name string) string {
```

Comment:
Not related to the original issue, just noticed that it is not in use, and since I touched this file, I cleaned it up a bit.